Modules/Core/Common/include/itkResourceProbe.hxx

```diff
@@ -143,7 +143,7 @@ ResourceProbe<ValueType, MeanType>::Stop()
   this->UpdateMinimumMaximumMeasuredValue(probevalue);
   this->m_TotalValue += probevalue;
   this->m_ProbeValueList.push_back(probevalue);
-  this->m_NumberOfStops++;
+  ++(this->m_NumberOfStops);
```

Comment:
OK to me, thanks, but we might eventually consider to just remove this `this->`, because it just points to its very own data member, `ResourceProbe::m_NumberOfStops`. Then we can also remove the parentheses that you're adding here  😃